app/core/views/cities/index.pug

```diff
@@ -1,9 +1,21 @@
-include ../_components/html
-
-+doc()
-  +head()
-  +body()
+doctype html
+html(lang='en')
+  head
+    meta(charset='UTF-8')
```

Comment:
These `meta`, `link` and `script` tags are now duplicated and will need to be updated in two places if anything changes. The reason for using a .pug mixin is to prevent exactly that. These should have been placed in the mixin so they can be used everywhere.

Most of these were already defined in the mixin: `/ExpressBoilerplate/app/core/views/_components/html.pug`